tests/components/image/common.yaml

```
@@ -5,32 +5,36 @@ image:
     dither: FloydSteinberg
   - id: transparent_transparent_image
     file: ../../pnglogo.png
-    type: TRANSPARENT_BINARY
+    type: BINARY
+    use_transparency: chroma_key
   - id: rgba_image
     file: ../../pnglogo.png
-    type: RGBA
+    type: RGB
+    use_transparency: alpha_channel
     resize: 50x50
   - id: rgb24_image
     file: ../../pnglogo.png
-    type: RGB24
-    use_transparency: yes
+    type: RGB
+    use_transparency: alpha_channel
```

```suggestion
    use_transparency: chroma_key
```
The combination "RGB" with "alpha_channel" is already being tested in the image just above this one